fluka.sh

```diff
@@ -4,40 +4,28 @@ tag: "2011-3.0-vmc2"
 source: https://gitlab.cern.ch/ALICEPrivateExternals/FLUKA.git
 requires:
   - "GCC-Toolchain:(?!osx)"
+build_requires:
+  - alibuild-recipe-tools
 env:
   FLUPRO: "$FLUKA_ROOT/lib"
   FC: "gfortran"
 prepend_path:
   PATH: "$FLUKA_ROOT/bin"
 ---
-#!/bin/bash -e
-
 export FLUPRO=$PWD
 export FC=gfortran
 
-rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ "$INSTALLROOT"
-cd "$INSTALLROOT"
+rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ "$BUILDDIR"
 make ${JOBS:+-j$JOBS}
```

Comment:
If we replace this line by
FVERSION=`gfortran --version | grep -i fortran | sed -e 's/.* //' | cut -d. -f1`
if [ $FVERSION -ge 10 ]; then
    echo "Fortran version $FVERSION"
# Redefine FC to contain the compiler, -fallow-argument-mismatch and the additional FFLAGS from FLUKA/src/config.mk
    make FC="gfortran -fallow-argument-mismatch -Wall -Waggregate-return -Wcast-align -Wline-truncation -Wno-conversion -Wno-integer-division -Wno-tabs -Wno-unused-dummy-argument -Wno-unused-function -Wno-unused-parameter -Wno-unused-variable -Wsystem-headers -Wuninitialized -Wunused-label -mtune=generic -msse2 -mfpmath=sse -fPIC -fexpensive-optimizations -funroll-loops -fstrength-reduce -fno-automatic -finit-local-zero -ffixed-form -fbackslash -funderscoring -fd-lines-as-code -frecord-marker=4 -fbacktrace -frange-check -fbounds-check -fdump-core -ftrapping-math -ffpe-trap=invalid,zero,overflow" ${JOBS:+-j$JOBS}
else
    make ${JOBS:+-j$JOBS}
fi
we will solve also the issue with gfortran 10.

Reply:
Of course, we can do it in a separate pull request.